validphys2/src/validphys/asy_exponents.py

```diff
@@ -0,0 +1,266 @@
+# -*- coding: utf-8 -*-
+"""
+Tools for computing and plotting effective exponents.
+"""
+from __future__ import generator_stop
```

Comment:
This is not needed for any current version of Python.

Reply:
Removed.